mysql/toddl.go

```diff
@@ -228,3 +232,38 @@ func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Forei
 	}
 	return spKeys
 }
+
+func cvtIndexes(conv *internal.Conv, spTableName string, srcTable string, srcIndexes []schema.Index, schemaIndexKeys map[string]bool) []ddl.CreateIndex {
+	var spIndexes []ddl.CreateIndex
+	for _, srcIndex := range srcIndexes {
+		var spKeys []ddl.IndexKey
+		for _, k := range srcIndex.Keys {
+			spCol, err := internal.GetSpannerCol(conv, srcTable, k.Column, true)
+			if err != nil {
+				conv.Unexpected(fmt.Sprintf("Can't map index key column name for table %s", srcTable))
+				continue
+			}
+			spKeys = append(spKeys, ddl.IndexKey{Col: spCol, Desc: k.Desc})
+		}
+		if srcIndex.Name == "" {
+			// Generate a random name if index name is empty in postgres
+			// since spanner requires globally unique non-empty index names.
+			b := make([]byte, 4)
+			if _, err := rand.Read(b); err != nil {
+				conv.Unexpected(fmt.Sprintf("Can't map index name for table %s", srcTable))
+				continue
+			}
+			srcIndex.Name = fmt.Sprintf("Index_%s_%x-%x", srcTable, b[0:2], b[2:4])
```

Comment:
I'm wondering if we need the complexity of random generation here.
Couldn't we just use 
```
srcIndex.Name = fmt.Sprintf("Index_%s", srcTable)
```
and then rely on ToSpannerIndexKey to add a unique id if needed?

Reply:
yes make sense to simplify this